frontend/src/components/Sidebar/DistrictMap.tsx

```diff
@@ -166,38 +160,33 @@ export default function DistrictMap(): JSX.Element {
     };
   }, [geodata, theme, t, formatNumber, dispatch]);
 
-  // TODO: district search for highlighting
-  /*
+  const polygonSeriesLength = (chartRef.current?.series.getIndex(0) as am5map.MapPolygonSeries)?.mapPolygons.length; //needed as trigger for the following effect
```

Comment:
When reloading the page and restoring the state from localStorage, this is needed to trigger the effect after the polygonSeries is populated with data.